### PR TITLE
Fixing several small documentation bugs

### LINKED ISSUE #316: link to protobuf 3 incorrect / not available
https://github.com/phenopackets/phenopacket-schema/blob/v2/docs/requirements.rst

### LINKED ISSUE #339: Wrong example code at "Interpretation" Candidate Genes [documentation]
The "[Candidate genes](https://phenopacket-schema.readthedocs.io/en/latest/interpretation.html#candidate-genes)" section of the Interpretation's examples has the wrong example (copied from the main example, but unchanged for this particular case). 
image

### PR DESCRIPTION
**Pull request by ielis**
@julesjacobsen 
The PR fixes several small inaccuracies in the documentation, mostly deviations from the protobuf schema.

Fixes #316, #339, #344, #350, #351

### CODE REVIEW

**Review by pnrobinson — CHANGES_REQUESTED** [review]
Looks fine but there is not reason for the change in interpretation/Miller syndrome

**Comment by ielis** [comment]
Hi @pnrobinson 
I will fix this soon, but there is one more thing I'd like to check. Should we italicize gene symbols? Currently, the documentation does not italicize human gene symbols.

**Comment by pnrobinson** [comment]
gene symbols should alsways be italicized!

File: docs/interpretation.rst [hunk 1]
```diff
@@ -150,7 +150,7 @@ with the hypothetical gene YFG42.
         id: "OMIM:263750"
         label: "Miller syndrome"
       genomicInterpretations:
-      - interpretationStatus: "CONTRIBUTORY"
+      - interpretationStatus: "CANDIDATE"
```
Comment (pnrobinson) [inline docs/interpretation.rst:153]: In this case the interpretationStatus should be CAUSAL, this was a published case!

**Review by pnrobinson — CHANGES_REQUESTED** [review]
One small change requested then we can merge!

File: docs/interpretation.rst (hunk 1 above)
Comment (pnrobinson) [reply docs/interpretation.rst:153]: Maybe it does not fit well with the description, but it si not wrong, I will merge

**Pull request merged**